scripts/create_registry_schema.py

```diff
@@ -0,0 +1,365 @@
+import argparse
+import pandas as pd
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    DateTime,
+    Boolean,
+    Index,
+    Float,
+)
+from sqlalchemy import ForeignKey, UniqueConstraint, text
+from sqlalchemy.orm import DeclarativeBase
+from dataregistry.db_basic import DbConnection, SCHEMA_VERSION
+from dataregistry.db_basic import _insert_provenance
+from dataregistry.schema import load_schema
+
+"""
+A script to create a schema.
+
+Both schemas have the same layout, containing six tables:
```

Comment:
"Both" maybe better now as "All"

Reply:
Yes, that text is out of date.   "Both" doesn't really enter into it anymore.